resources/idlharness.js

```diff
@@ -1684,6 +1691,9 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
     for (var i = 0; i < this.members.length; i++)
     {
         var member = this.members[i];
+        if (member.untested) {
```

Comment:
Yes, the style in this file is inconsistent, so I tried to be locally consistent.